RetailCoder.VBE/Navigation/CodeExplorer/CodeExplorerViewModel.cs

```diff
@@ -80,7 +80,7 @@ private void ParserState_StateChanged(object sender, EventArgs e)
         {
             Debug.WriteLine("CodeExplorerViewModel handles StateChanged...");
             IsBusy = _state.Status == ParserState.Parsing;
-            if (_state.Status != ParserState.Parsed)
+            if (_state.Status != ParserState.Ready)
```

Comment:
Not sure.. we need to test @Vogel612's upcoming changes, to see whether we have a `Declaration` for modules that can't be parsed, and whether we have `UserDeclarations` at all, when any module produces a parser error.